installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -16,76 +16,111 @@ include::modules/oc-mirror-about.adoc[leveloffset=+1]
 
 // Installing the oc-mirror OpenShift CLI plugin
 include::modules/oc-mirror-installing-plugin.adoc[leveloffset=+2]
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../cli_reference/openshift_cli/extending-cli-plugins.adoc#cli-installing-plugins_cli-extend-plugins[Installing and using CLI plugins]
-
 // Configuring credentials that allow images to be mirrored
 include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
 
 // Creating the image set configuration
 include::modules/oc-mirror-creating-image-set-config.adoc[leveloffset=+1]
 
+// Image set configuration parameters
+include::modules/oc-mirror-imageset-config-params.adoc[leveloffset=+2]
+
+// Image set configuration examples
+include::modules/oc-mirror-image-set-config-examples.adoc[leveloffset=+1]
+
+[id="setting-incremental-mirroring"_{context}]
+=== Setting up for incremental mirroring
+
+In most cases, you are mirroring images to install, upgrade, and maintain {product-title} in a disconnected environment. Because the volume of the image set needed to install or upgrade {product-title} can be substantial, oc-mirror keeps metadata about previously mirrored images in its storage backend in order to reduce the volume of the following image sets to the recent images only.
+
+Therefore, you must specify a storage backend in the image set configuration file. The oc-mirror plugin stores metadata in this storage backend during image set creation. You can specify the following storage backends:
+
+* A local directory 
++
+[NOTE]
+====
+This is not recommended for enterprise use cases as it does not scale as well as the registry-based storage backend.
+====
++
+[source,terminal]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:                                                    
+  local:
+    path: <oc_mirror_workspace>
+----
+
+* A registry that supports Docker v2-2
++
+[source,terminal]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:                                                    
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata                 
+    skipTLS: false
+----
+
+When mirroring content to a given location on your mirror registry with the oc-mirror plugin, consider the following:
+- Ensure you always use the same storage backend and the same image set configuration for a given mirror registry location, so that the metadata remains consistent.
+- Do not attempt to push or prune images of that mirror registry location manually, without using oc-mirror.
+- Do not delete or modify the metadata that is generated by the oc-mirror plugin.
+- Keep a backup of the storage backend when mirroring content to a given location on your mirror registry with the oc-mirror plugin.
```

Comment:
Looks like this still needs a blank space before the list:
```suggestion
When mirroring content to a given location on your mirror registry with the oc-mirror plugin, consider the following:

- Ensure you always use the same storage backend and the same image set configuration for a given mirror registry location, so that the metadata remains consistent.
- Do not attempt to push or prune images of that mirror registry location manually, without using oc-mirror.
- Do not delete or modify the metadata that is generated by the oc-mirror plugin.
- Keep a backup of the storage backend when mirroring content to a given location on your mirror registry with the oc-mirror plugin.
```